pandas/_typing.py

```diff
@@ -219,6 +219,10 @@
 PositionalIndexer = Union[ScalarIndexer, SequenceIndexer]
 PositionalIndexerTuple = Tuple[PositionalIndexer, PositionalIndexer]
 PositionalIndexer2D = Union[PositionalIndexer, PositionalIndexerTuple]
+if TYPE_CHECKING:
+    TakeIndexer = Union[Sequence[int], Sequence[np.integer], npt.NDArray[np.integer]]
+else:
+    TakeIndexer = Union[Sequence[int], Sequence[np.integer], np.ndarray]
```

Comment:
if not type checking, I guess this could be None or Any to reduce noise?

Reply:
Changed it.